### ISSUE TITLE
Lint to check post messages are translatable is throwing false positives

### ISSUE THREAD

**Issue by luisg123v** [OCA/pylint-odoo#187]
The lint `translation-required` is intended to detect not-translatable messages withing methods like `message_post`, like in this code:
```
self.message_post(body='some message'))
```
And suggest using the function `_()` to make them translatable.

However, codes like the following are triggering false positives:
```
self.message_post(
    body=_('Error during the process: %s') % message,
    subtype='account.mt_invoice_validated')
```

Showing the following error message:
```
l10n_mx_edi_payslip/models/hr_payslip.py:264: [C8107(translation-required), HrPayslip.l10n_mx_edi_log_error] String parameter on "message_post" requires translation. Use body=_((_('Error during the process: %s')) % (message))
```

Not sure if this has anything to do with `message_post` and `_(...)` being located on different lines.
